File: Software/CapstoneProject/CapstoneProject.Model/Entities/Job.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CapstoneProject.Model.Entities
{
    /// <summary>
    /// A Job belongs to a department and is assigned to Employees
    /// </summary>
    public class Job : Base
    {
        /// <summary>
        /// The Jobs unique identifier
        /// </summary>
        public int ID { get; set; }

        /// <summary>
        /// The Name of the Job
        /// </summary>
        public string Name { get; set; }
    }
}

File: Software/Website/Cryptech/src/app/models/job.ts
import { Base } from './base';
import { Department } from './department';

/**
 * Job Class
 * @class
 * @extends Base
 */
export class Job extends Base {
  /**
   * @property {number} id - The Jobs ID
   */
  id: number;

  /**
   * @property {string} name - The name of the Job.
   */
  name: string;
}
